packages/features/schedules/components/Schedule.tsx

```diff
@@ -383,21 +383,39 @@ const CopyTimes = ({
       <div className="p-2">
         <p className="h6 text-emphasis pb-3 pl-1 text-xs font-medium uppercase">{t("copy_times_to")}</p>
         <ol className="space-y-2">
+        <li key="select all">
+                <label className="text-default flex w-full items-center justify-between">
```

Comment:
Is it just me or is this spacing off?

Reply:
yes spacing is definitely off here, @a0m0rajab can you fix that?

Reply:
Thank you for that! Just fixed it 👍